aws/db.tf

```diff
@@ -80,3 +80,12 @@ module "db" {
 
   tags = local.tags
 }
+
+resource "aws_route53_record" "db_private_dns" {
+  zone_id = aws_route53_zone.internal[0].zone_id
```

Comment:
There is a condition for internal zone, so it is important to check it here too

Reply:
Condition added